.changeset/dry-paws-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/utils": minor
+---
+
+add @fuel-ts/utils/assets-utils package
```

Comment:
It looks more like a directory/script. 🤔

Reply:
Hmm.. thought of it as a package/mini-package since I put it under a separate export path. 

Directory makes sense tho lol!

Reply:
After you move it around, the description may naturally morph with it.